edx_notifications/server/api/consumer.py

```diff
@@ -211,6 +212,25 @@ def post(self, request, msg_id):
         return Response({}, status.HTTP_200_OK)
 
 
+class MarkNotifications(AuthenticatedAPIView):
+    """
+    Mark all the user notifications as read
+    """
+
+    def post(self, request):
+        """
+        HTTP POST Handler which is used for such use-cases as 'mark as read'
+        """
+        try:
+            # this will raise an ItemNotFoundError if the user_id/msg_id combo
+            # cannot be found
+            mark_all_user_notification_as_read(request.user.id)
```

Comment:
We should probably allow for a POST variable to say whether all UserNotifications should be marked as read OR unread. This isn't the current use-case, but I'd anticipate we might want to support this in the future.